app/data_model/answer_store.py

```diff
@@ -37,7 +37,7 @@ def matches_dict(self, answer_dict):
         Check to see if a dict describes an answer the same as this object.
 
         :param answer_dict: A dict providing
```

Comment:
I know its existing but `:param answer_dict: A dict providing`, that doesn't make sense?

Reply:
A dictionary representation of the answer?